Robust.Shared/Localization/LocHelper.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Text;
+using Linguini.Bundle.Errors;
+using Linguini.Syntax.Parser.Error;
+
+namespace Robust.Shared.Localization
+{
+    public static class LocHelper
```

Comment:
```suggestion
    internal static class LocHelper
```

I'd say this is a pretty internal helper method that content shouldn't concern itself with.